### ISSUE TITLE
Crime Crackers - Random Freezing / Music Not Looping

### ISSUE THREAD

**Issue by voidflower-xcii** [libretro/swanstation#60]
Crime Crackers seems to be having some odd timing issues. Sometimes it will lock-up at the title screen and sometimes it won't. Sometimes after you push start is will freeze at a black screen and sometimes it won't. Same deal after you select a save slot. I've only gotten to play about 30 minutes of the gameplay so I can't comment much on that at the moment.

Upon actually getting into the gameplay however, it quickly becomes apparent the music is not looping when it should, leaving you with silence. When you enter certain areas a different track sometimes starts, but I'm assuming this would have been an intended track change.

I've confirmed the MD5/SHA-1 hashes against redump.org. (http://redump.org/disc/5729/)

**Comment by DarthMew** [comment]
I've tried booting the game a few times, but I'm not getting the freezes you've described in your post.
Is there anything special you're doing, like being very quick with going through the process or something?

Also, could you tell me what version of SwanStation you're using at the moment (you can see the version number and git hash when you're in the quick menu when you have loaded up a game)?

**Comment by voidflower-xcii** [comment]
The first time it happened I was letting the intro text scroll play for a bit as I listened to the music for a minute or so before hitting start and the game promptly froze on the title screen never displaying the "Push Start Button" text or playing the music. I re-launched the game, this time mashing my way past and getting to the save slot screen. I took another minute or two on this screen and when I selected a save slot it froze on the fade out. The third time I mashed my way through every screen and got into the gameplay where I then noticed the music bug.

Eventually I quit and re-launched the game again later, and after mashing through both screens again I actually managed to make a note from the music on the save screen hang indefinitely into the cutscene and gameplay. Re-launched _again_ and it froze on the fade out from the save screen, and finally again I got into the gameplay fine. It seems very random to me.

I should have probably mentioned, what CPU mode are you launching it with? I've only tried it with Interpreter at the moment.

![version](https://user-images.githubusercontent.com/120467626/218942843-ca2f8e51-fffa-473d-bbd8-18987729b69a.png)

**Comment by DarthMew** [comment]
Try and see if it happens when using default settings (CPU recompiler, no overclock, 1x resolution scale etc.).
Also check what video driver you're using, and if you haven't enabled any incompatible settings in RetroArch itself, like runahead.

**Comment by voidflower-xcii** [comment]
I've turned recompiler back on and never messed with any resolution or overclocking settings. The game seemed to function normally on the menus and the music even looped again. However after about 10-15 minutes of gameplay the music stopped looping again for a brief moment before changing to a completely different track, and then when trying to use a door with a dialog option the game hard locked again. I have the video driver set to Vulkan currently.

![Crime Crackers (Japan)-230215-075154](https://user-images.githubusercontent.com/120467626/219046671-8f3df8e2-00ef-413e-9eb6-69c4f68d1cd1.png)

**Comment by DarthMew** [comment]
Yeah the softlock seems to be very random, because I've been able to reproduce it very inconsistently in SwanStation, and in upstream as well.
It might be a very weird CD read timing issue.

Did this issue always occur for you, or is it something that started happening recently?

**Comment by voidflower-xcii** [comment]
This was my first time checking this title out, so I'm not sure if it's a long-standing problematic game unfortunately.
